test/system/500-networking.bats

```diff
@@ -1039,4 +1039,19 @@ EOF
     run_podman network rm $netname
 }
 
+# 2024-07-23 moved from 505 pasta tests because it can't run in parallel
+@test "IPv4 default route assignment with pasta" {
```

Comment:
I meant `Podman unshare --rootless-netns with Pasta`, this one here should work fine in parallel
(in theory at least it is not clear to me why I am seeing several flakes locally)

Reply:
ack, will redo in next push.

This push adds `--name` to pasta podman run, in hopes that it helps identify stuck containers in the leak check.